webserver/Models/RealEstate.cs

```diff
@@ -1,24 +1,33 @@
 ﻿namespace webserver.Models;
-
 public class RealEstate {
 
     public int Id { get; set; }
     public string Name { get; set; } = string.Empty;
     public string Address { get; set; } = string.Empty;
     public decimal Price { get; set; }
 
-    public bool rentable { get; set; }
+    public string CompanyId { get; set; } = string.Empty;
+    // Navigation property
+    [Newtonsoft.Json.JsonIgnore]
+    public Company? OwnerCompany { get; set; }
```

Comment:
This is excluding the field from the output, I got what you did now.
But can you do the opposite way?

I mean, explicitly say what will be in the output, instead of excluding it.

Reply:
[JsonProperty] it is